app/grants/router.py

```diff
@@ -88,59 +93,161 @@ def get_queryset(self):
 
         return queryset
 
-    @action(detail=False)
-    def report(self, request):
-        return Response({'error': 'reports temporarily offline'})
 
     @action(detail=False)
-    def report_real(self, request):
-        """Generate Grants report for an ethereum address"""
+    @ratelimit(key='ip', rate='5/s')
+    def contributions_rec_report(self, request):
+        """Genrate Grantee Report for an Grant"""
```

Comment:
Consider: Would it be worth including details of the expected request format here?

Reply:
I can throw it in a readme file ! Would you rather we add it here ?

Reply:
We can assume (or can we?) that users who will be accessing our end-points will likely check the code, I think that it would be useful to include these details inline, and later we can look to create documentation from the code itself? If theres a better place we can leave these details though, we can explore that, but I still feel inline would be a safe approach?